File: 2ndSemester/AlgorithimsAndDataStructuresII/Practice/Sums/ArithmeticProgression.java
public class ArithmeticProgression{
	/*
	  a -> termo inicial
	  b-> razao
	  n->limite superior
	*/
	
	public static double sumAP(double a, double b, int n){
		double sum = 0;
		for(int i = 0; i <= n; i++){
			sum += a + (b * i);
			System.out.println("Current value  = " +  (a +(b * i)));
		}

		return sum;
	}
	public static void main(String[] args){
		System.out.println("Sum = " + sumAP(1, 3, 4));
	}
}

File: 2ndSemester/AlgorithimsAndDataStructuresII/Practice/Sums/arithmeticProgression.c
#include <stdio.h>
/*
	a-> Termo inicial
	b-> razao
	n-> limite superior
*/
double sumPA(double a, double b, int n){
	double sum = 0;
	for(int i = 0; i <= n; i++){
		sum += a + (b * i);
		printf("\nCurrent value = %g",a + (b * i));
	}
	return sum;
}

int main(){
	printf("Sum = %g\n", sumPA(1,3,4));
	return 0;
}
